projects/common/src/color/color-palette.ts

```diff
@@ -16,6 +16,10 @@ export class ColorPalette {
       return [...this.basisColors];
     }
 
+    if (count === 1) {
+      return [this.basisColors[0]];
```

Comment:
What was the previous behavior? Was it giving a mid range value? The modified test indicates it was doing what this is doing.

Reply:
The previous behavior was always outputting the value as `rgb(0, 0, 0)` for `count = 1`, irrespective of the color palette

That's why I changed the test case, since it was using `rgb(0, 0, 0)` as first color in the color palette, which made it pass erroneously

Reply:
gotcha, sneaky! also is that GH doing the color rendering? `#FF0000` `rgba(255, 0, 0, 0.5)` `rgb(255,0,0)` Neat, learned something new